include/pmacc/mappings/simulation/Filesystem.cpp

```diff
@@ -31,15 +28,13 @@
 
 namespace pmacc
 {
-    template<unsigned DIM>
-    void Filesystem<DIM>::createDirectory(const std::string dir) const
+    void Filesystem::createDirectory(const std::string dir) const
     {
         /* does not throw if the directory exists or has been created */
```

Comment:
Would you mind adding this minor clarification while you're touching this file?
```suggestion
        /* using `create_directories` instead of `create_directory` because the former does not throw if the directory exists or has been created */
```

Reply:
fixed, thx